libs/elements/layout/buttons/src/lib/normal-button/normal-button.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+// imported the input
+@Component({
+  selector: 'elewa-normal-button',
+  templateUrl: './normal-button.component.html',
+  styleUrls: ['./normal-button.component.scss'],
+})
+export class NormalButtonComponent {
+ 
+  @Input() buttonText: string = '';
```

Comment:
Good Job. But the issue spec was updated and this will now require a type of buttonData. Check the issue description.

Reply:
It can require a single ButtonData input OR the class can implement ButtonData. The second option is in fact cleaner in this case. Just make sure that the implemented class members are marked as @Input()

Reply:
There is no @Output event that emits the OnClick event

Reply:
Apologies for that, i have managed to push new updates, that will allow the Onclick event